tests/e2e/cucumber/features/smoke/upload.feature

```diff
@@ -77,7 +69,17 @@ Feature: Upload
       | child.txt |
     And "Alice" logs out
 
+  @webkit-skip
+  Scenario: upload empty folder
+    When "Alice" uploads the following resources
+      | resource | type   |
+      | FOLDER   | folder |
+    Then following resources should be displayed in the files list for user "Alice"
+      | resource |
+      | FOLDER   |
+    And "Alice" logs out
```

Comment:
I'm just seeing this now - how is this test successful when https://github.com/opencloud-eu/web/issues/821 is not yet implemented? 🤔 Is it because there is actually a `.gitkeep` file in `FOLDER`?

Reply:
yes, true because `.gitkeep` in the folder. I'll delete this step